stylesheets/pages/home.css

```diff
@@ -10,7 +10,38 @@ body {
 }
 
 .home-hero {
-  cursor: pointer;
+  position: relative;
+}
+
+.hero-img {
+  opacity: 0.5;
+}
+
+.hero-content {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;

```

Comment:
Extract these properties onto a utils class.

Reply:
Thanks for pointing `_` class. It was a mistake. Refactored it.